frappe/core/doctype/package_release/package_release.py

```diff
@@ -7,15 +7,19 @@
 import os
 import subprocess
 
+
 class PackageRelease(Document):
 	def set_version(self):
 		# set the next patch release by default
+		from frappe.query_builder.functions import Max
+		from frappe.query_builder import Field
+
 		if not self.major:
-			self.major = frappe.db.max('Package Release', 'major', dict(package=self.package))
+			self.major = frappe.qb.from_("Package Release").where(Field("package") == self.package).select(Max("major")).run()[0][0] or 0
 		if not self.minor:
-			self.minor = frappe.db.max('Package Release', 'minor', dict(package=self.package))
+			self.minor = frappe.qb.from_("Package Release").where(Field("package") == self.package).select(Max("minor")).run()[0][0] or 0
 		if not self.patch:
-			self.patch = frappe.db.max('Package Release', 'patch', dict(package=self.package)) + 1
+			self.patch = frappe.qb.from_("Package Release").where(Field("package") == self.package).select(Max("patch")).run()[0][0] or 1
```

Comment:
This doesn't seem to be equivalent @Aradhya-Tripathi

Reply:
Hey, @gavindsouza both the queries 
```'SELECT MAX('minor') FROM "tabPackage Release" WHERE "package"=\'test package\'''```
and
```'SELECT MAX(minor) FROM "tabPackage Release" WHERE "package"=\'test package\'''```
are executed in a similar fashion and "minor" is considered to be the same in both.